include/declarations.h

```diff
@@ -9,6 +9,123 @@ extern u8 gUnknown_08013B64[];
 extern const u8 gUnknown_08014D70[];
 extern const u16 gUnknown_08014D82[]; // GS1_gameover_message_data_table
 extern const u8 gUnknown_08013B70[0x400];
+
+extern u8 gUnknown_08015100[0x78];
```

Comment:
These should all use the `ExaminationData` struct. I mentioned this before in #46

Reply:
since you are working on reorganizing data on your branch i would leave these as-is for now and revisit once the reorganization is done